packages/table/src/table.tsx

```diff
@@ -542,6 +542,7 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
             this.bodyElement,
             this.grid,
         );
+        this.locator = locator;
 
         const viewportRect = locator.getViewportRect();
         this.setState({ locator, viewportRect });
```

Comment:
ok so why is `locator` stored in state at all at this point? seems like `viewportRect` should be sufficient.